src/v0/destinations/tiktok_ads/networkHandler.js

```diff
@@ -15,6 +15,9 @@ const responseHandler = (destinationResponse) => {
       destinationResponse,
     };
   }
+  if (code === 40100) {
+    throw new RetryableError(`Request failed with status: ${code}`, 500, destinationResponse);
+  }
```

Comment:
`"code":40100,"message":"Too many requests. Please retry in some time`

status code should be 429

Reply:
Addressed. Put 500 in lieu of this being some server issue over at Tiktok, but since it is retryable, it should be 429.